### PR TITLE
Direct connection: Interface and VLAN options

### PR DESCRIPTION
**Pull request by lukas2511**
### Fixes:

* [x]  Add Interface option to direct connections
* [x]  Add VLAN option to direct connections

### CODE REVIEW

File: peering/migrations/0079_directpeeringsession_vlan.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.7 on 2021-09-20 13:29
+
+from django.db import migrations
+import net.fields
+
+
+class Migration(migrations.Migration):
```
Comment (jamesditrapani) [inline peering/migrations/0079_directpeeringsession_vlan.py:7]: Let's squash both migration files into one.

File: peering/models/models.py [hunk 2]
```diff
@@ -524,6 +525,8 @@ class DirectPeeringSession(BGPSession):
     router = models.ForeignKey(
         "Router", blank=True, null=True, on_delete=models.SET_NULL
     )
+    interface = models.CharField(max_length=200, blank=True)
```
Comment (jamesditrapani) [inline peering/models/models.py:528]: You will need to update the test cases in peering/tests with the new fields 😀

**Review by jamesditrapani — CHANGES_REQUESTED** [review]
Looks like black & isort pre merge checks have failed. Can you please run black & isort to ensure the new code is within specification. Thank you for your contribution!

**Comment by lukas2511** [comment]
I merged the db migration and added the fields to the tests. isort and black are also looking good on my end, hopefully CI thinks the same :D

File: peering/models/models.py (hunk 2 above)
Comment (jamesditrapani) [inline peering/models/models.py:528]: The only other thing that I would point out is to ensure the documentation is updated for the DirectPeeringSession model here https://github.com/peering-manager/peering-manager/blob/435f75e87cbb49941e10c7bd5a4663ae398867bc/docs/models/peering/directpeeringsession.md

Sorry, forgot to include this in the previous review! :)

**Review by jamesditrapani — CHANGES_REQUESTED** [review]

**Comment by gmazoyer** [comment]
I'd advise to use the `net.models.Connection` class instead of duplicating fields specifically for direct peering session. The `Connection` model was introduced to do just this.

**Comment by jamesditrapani** [comment]
Hi @lukas2511, did you want to have a look at moving this to use `net.models.Connection` as @gmazoyer suggested? If not I can jump into your branch and do so :)

**Comment by lukas2511** [comment]
> Hi @lukas2511, did you want to have a look at moving this to use `net.models.Connection` as @gmazoyer suggested? If not I can jump into your branch and do so :)

Unfortunately I didn't find time to look into this. My simple solution was working fine for me and I didn't really want to invest any more work.

**Pull request closed without merging**